benchmarks/benchmarks/stats.py

```diff
@@ -56,7 +56,99 @@ def time_ttest_ind_diff_var(self):
         stats.ttest_ind(self.a, self.c, equal_var=False)
 
 
+class DistributionsAll(Benchmark):
+    # all distributions are in this list. A conversion to a set is used to
+    # remove duplicates that are files under `distcont` and `distdiscrete`.
```

Comment:
```suggestion
    # remove duplicates that are included in both `distcont` and `distdiscrete`.
```
"files"?
Do you remember offhand which were included in both places?

Reply:
```
from scipy.stats._distr_params import distcont, distdiscrete

import collections

dists = [d[0] for d in distcont + distdiscrete]

print([item for item, count in collections.Counter(dists).items() if count > 1])
['gengamma', 'kappa4', 'truncnorm', 'hypergeom', 'nbinom']
```

Reply:
They are not in both lists; all of these are included multiple times in the same (respective) list. E.g. `gengamma` appears twice in `distcont`.  Please adjust the comment to make that more clear.
```python3
distcont = set((d[0] for d in distcont))
distdiscrete = set((d[0] for d in distdiscrete))
distcont.intersection(distdiscrete)  # empty set
```